### DISCUSSION TITLE
How do I override spammy addresses?

### DISCUSSION THREAD

**Discussion by krangor** [buttondown/discussions#54] [Q&A]
Hi! My uncle-in-law recently tried to sign up for my newsletter and got rejected for being spammy or invalid. He is, I assure you, neither of those things, just an uncle. I tried to enter his address manually but I was also rejected (see screenshot). Is there a way to override this rejection? Also, are you sure your filters are doing their job and not producing false positives?
![image](https://github.com/user-attachments/assets/e2debcf5-29b1-46da-9d92-9d5084f47353)

**Comment by jmduke** [comment]
It is definitely _entirely possible_ this is a false positive. Mind emailing me the address in question (concierge@buttondown.com)?

**Reply by krangor** [reply]
Also, I looked at the flow for spammy addresses and it seems... a little problematic? You give the option to "Contact Author," which is just handing over my un-obscured email address to the people the algorithm has deemed the sketchiest, which doesn't feel great! And if there's no way to override the flag, what good does having them contact me do?
